### ISSUE TITLE
 'Unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'"

### ISSUE THREAD

**Issue by ilikescience** [AtomLinter/linter-stylelint#116]
Full error:
`Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'".`

Stack Trace:
```
EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'".
    at Function (native)
    at installColorSpace (/Users/strom/.atom/packages/linter-stylelint/node_modules/stylelint/node_modules/colorguard/node_modules/pipetteur/node_modules/onecolor/one-color-all-debug.js:97:32)
    at Object.<anonymous> (/Users/strom/.atom/packages/linter-stylelint/node_modules/stylelint/node_modules/colorguard/node_modules/pipetteur/node_modules/onecolor/one-color-all-debug.js:218:1)
    at Module._compile (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:103:30)
    at Object.defineProperty.value [as .js] (/Applications/Atom.app/Contents/Resources/app.asar/src/compile-cache.js:208:21)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/Users/strom/.atom/packages/linter-stylelint/node_modules/stylelint/node_modules/colorguard/node_modules/pipetteur/pipetteur.js:2:13)
    at Module._compile (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:103:30)
    at Object.defineProperty.value [as .js] (/Applications/Atom.app/Contents/Resources/app.asar/src/compile-cache.js:208:21)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/Users/strom/.atom/packages/linter-stylelint/node_modules/stylelint/node_modules/colorguard/lib/colorguard.js:5:17)
    at Module._compile (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:103:30)
    at Object.defineProperty.value [as .js] (/Applications/Atom.app/Contents/Resources/app.asar/src/compile-cache.js:208:21)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/Users/strom/.atom/packages/linter-stylelint/node_modules/stylelint/node_modules/colorguard/index.js:1:97)
    at Module._compile (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:103:30)
    at Object.defineProperty.value [as .js] (/Applications/Atom.app/Contents/Resources/app.asar/src/compile-cache.js:208:21)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:50:27)
```

On Atom v1.5.4
Using linter-stylelint v1.13.1
Mac OSX 10.11.3

**Comment by steelbrain** [comment]
See https://github.com/stylelint/stylelint/issues/891

**Comment by Arcanemagus** [comment]
v2.0.0 has been published which updates `stylelint` to v5.0.0.
